Source/scrollrt.cpp

```diff
@@ -2393,12 +2393,12 @@ void DrawMain(int dwHgt, BOOL draw_desc, BOOL draw_hp, BOOL draw_mana, BOOL draw
 		}
 	}
 
-	/// ASSERT: assert(ysize >= 0 && ysize <= 480);
+	/// ASSERT: assert(ysize >= 0 && ysize <= SCREEN_HEIGHT);

```

Comment:
You shouldn't be changing any of the asserts. Reason is that the expression is translated directly to a string so whatever is listed matches the debug release. By doing this the string will say `SCREEN_HEIGHT` instead of `480`.

Reply:
But... won't the assert trigger more often when the value isn't updated? (And no longer be useful in actual debugging? [e.g. when SCREEN_HEIGHT > 480.])

Reply:
Devilution tries to stick to the original source, changes like that will be made in DevilutionX

Reply:
Correct, consider assertions more of a bonus feature, since they weren't in any release build. Currently they are all commented out and are mostly just for documentation purposes. DevilutionX can update them accordingly.

Reply:
Thanks that makes things super clear to me. -- I didn't realize there was a DevilutionX.  I'll take a look.